managed/provider.go

```diff
@@ -157,6 +157,7 @@ func (p *provider) GetResources(_ context.Context) (map[string]tfsdk.ResourceTyp
 		"ybm_metrics_exporter":                   resourceMetricsExporterType{},
 		"ybm_associate_metrics_exporter_cluster": resourceAssociateMetricsExporterClusterType{},
 		"ybm_integration":                        resourceIntegrationType{},
+		"ybm_database_query_logging":             resourceDbQueryLoggingType{},
```

Comment:
I suggest we keep this resource under a FF
@posriniv pg logs is GA wrt to backend and UI. Is it a good idea to keep this under a FF. Afaik, pg logs is under FF for CLI. @bhupendray-yb can you please confirm?

Reply:
Yes, in CLI it is under a feature flag.